sdks/python/apache_beam/internal/apiclient.py

```diff
@@ -225,11 +226,18 @@ def __init__(self, packages, options, environment_version):
       self.proto.sdkPipelineOptions = (
           dataflow.Environment.SdkPipelineOptionsValue())
 
-      for k, v in sdk_pipeline_options.iteritems():
-        if v is not None:
-          self.proto.sdkPipelineOptions.additionalProperties.append(
-              dataflow.Environment.SdkPipelineOptionsValue.AdditionalProperty(
-                  key=k, value=to_json_value(v)))
+      options_dict = {k: v
```

Comment:
The `options_dict` is used for the options passed to the service, not for display data. This is the original behavior of the service.
In the code below, for display data, all items are included as long as they are not equal to their default value.